app/src/main/java/de/tum/in/tumcampusapp/auxiliary/DrawerMenuHelper.java

```diff
@@ -94,6 +99,12 @@ public void populateMenu(Menu navigationMenu) {
                              .setIntent(new Intent(mContext, item.activity));
             }
         }
+        SubMenu appMenu = navigationMenu.addSubMenu("App");
```

Comment:
This needs to be translated using the Strings.xml.

Reply:
I'd just leave the submenu unnamed...

Reply:
Just checked the google apps and others: neither use a submenu name, so we might as well leave it out then.

Reply:
I agree, i first tried it that way, however leaving the subname empty will create a nasty space between the divider and the settings submenu entry. 
Already googled to find a solution for that, unfortunately this seems to be possible only by inflating a menu.xml and using a group.
We could move the submenus into a xml file, inflate it and add the entries programmatically. What do you think?

Reply:
Is it not possible to directly add an entry to the menu without a submenu? 
If not, then a inflated .xml should be fine

Reply:
I think you can just [add](https://developer.android.com/reference/android/view/Menu.html#add(int,%20int,%20int,%20java.lang.CharSequence)) the item with another groupid, which creates a divider

Reply:
```
use NONE if an item should not be in a group.
```

Reply:
@pfent you were right, worked by adding it directly to the navmenu